includes/conditions.php

```diff
@@ -42,6 +42,10 @@ public static function compare( $left_value, $right_value, $operator ) {
 				return false !== array_search( $left_value, $right_value );
 			case '!in':
 				return false === array_search( $left_value, $right_value );
+			case 'contains':
```

Comment:
It's same to 'in' '!in', just add the 'contain' cases:
```
case 'in':
case 'contains':
```

Reply:
it checks the other way around